common/reviews/api/typings-generator.api.md

```diff
@@ -54,9 +54,10 @@ export class TypingsGenerator {
     generateTypingsAsync(): Promise<void>;
     // (undocumented)
     protected _options: ITypingsGeneratorOptions;
+    registerDependency(target: string, dependency: string): void;
     // (undocumented)
     runWatcherAsync(): Promise<void>;
-}
+    }
```

Comment:
This is... odd

Reply:
Indeed I wonder if I change it back and rebuild it, if it will leave it be? I'll try that.

Reply:
That fixed it. Very odd indeed.

Reply:
I think it is an API Extractor bug. Really we should invoke something like Prettier on the transformed expressions. Feel free to open an issue.